### PR TITLE
Fix problem with adding sonata-user to symfony recipes

### PR DESCRIPTION
**Pull request by aLkRicha**
I am targeting this branch, because no bc.

## Changelog

```markdown
### Changed
- use `symfony/security-core` and `symfony/security-acl` instead of `symfony/security`
```

## Subject
Fixing this [one](https://symfony.sh/r/github.com/symfony/recipes/332)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,11 +25,12 @@
...
-        "symfony/form": "^2.8 || ^3.2",
-        "symfony/http-foundation": "^2.8 || ^3.2",
-        "symfony/security": "^2.8 || ^3.2",
-        "symfony/translation": "^2.8 || ^3.2"
+        "symfony/console": "^2.8 || ^3.2 | ^4.0",
+        "symfony/form": "^2.8 || ^3.2 | ^4.0",
+        "symfony/http-foundation": "^2.8 || ^3.2 | ^4.0",
+        "symfony/security-core": "^2.8 || ^3.2 | ^4.0",
+        "symfony/security-acl": "^2.8 || ^3.2 | ^4.0",
+        "symfony/translation": "^2.8 || ^3.2 | ^4.0"
```
Comment (greg0ire) [inline composer.json]: You changed way more things that you said you changed.

Comment (aLkRicha) [reply composer.json]: @greg0ire yes, sorry, add one more line to changes

**Comment by greg0ire** [comment]
There should be 2 commits if there are 2 changes. And ideally, 2 PRs :P

File: composer.json [hunk 2]
```diff
@@ -28,7 +28,8 @@
         "symfony/console": "^2.8 || ^3.2",
         "symfony/form": "^2.8 || ^3.2",
         "symfony/http-foundation": "^2.8 || ^3.2",
-        "symfony/security": "^2.8 || ^3.2",
+        "symfony/security-acl": "^2.8 || ^3.0",
```
Comment (greg0ire) [inline composer.json]: why 3.0 for this one, 3.2 for the other?

Comment (greg0ire) [reply composer.json]: Oh I know, it simply does not exist for acl

Comment (aLkRicha) [reply composer.json]: @greg0ire because acl hase no 3.2 version

**Pull request merged**